package.json

```diff
@@ -1,51 +1,21 @@
 {
 	"name": "directus-monorepo",
 	"private": true,
-	"workspaces": [
-		"./app",
-		"./api",
-		"./packages/*"
-	],
 	"scripts": {
+		"build": "pnpm --recursive --filter '!docs' run build",
```

Comment:
as we're using `test` instead of `run test` in the "test" script below, should we make them consistent across the board (all with `run`, or all without `run` etc)?

same question for `run` in https://github.com/directus/directus/pull/18122/files#diff-a9d4fab00e8a7db561d841247150ac84dbf3c80ac94f96c4969bc4b66417eb4eR48

Reply:
Good question! For the `run` commands my goal was indeed to make it consistent 👍 For the test scripts I'm relying on the [`pnpm test`](https://pnpm.io/cli/test) command which I think makes it a bit more obvious.